packages/core-mobile/app/screens/rpc/components/shared/DetailSectionView.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import { Text, TouchableOpacity, View } from '@avalabs/k2-mobile'
+import { DetailSection } from '@avalabs/vm-module-types'
+import { Row } from 'components/Row'
+import TokenAddress from 'components/TokenAddress'
+import { bigIntToString, TokenUnit } from '@avalabs/core-utils-sdk'
+import { useSelector } from 'react-redux'
+import { selectAvaxPrice } from 'store/balance'
+import { selectSelectedCurrency } from 'store/settings/currency'
+import { useApplicationContext } from 'contexts/ApplicationContext'
+import { NodeID } from 'components/NodeID'
+import { getDateInMmmDdYyyyHhMmA } from 'utils/date/getDateInMmmDdYyyyHhMmA'
+
+export const DetailSectionView = ({
+  detailSection,
+  onPressDataItem
```

Comment:
I don't think you need to pass onPressDataItem

Reply:
but then we need to have a navigation logic in the subview component. I'd like to have navigation logics only in the screen components.